src/main/plugin/dcat-ap/schema/dcat.xsd

```diff
@@ -327,6 +340,26 @@
     </xs:sequence>
   </xs:complexType>
 
+
+  <!-- dcat:DatasetSeries -->
+  <xs:complexType name="DatasetSeries_type">
+    <xs:sequence>
+      <xs:element name="DatasetSeries">
+        <xs:complexType>
+          <xs:complexContent>
+            <xs:extension base="dcat:Dataset_type">
+              <xs:sequence>
+                <xs:element ref="dct:first" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="dct:last" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="dct:next" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
I believe these elements should be in the dcat namespace.   
As far as I understood, the `dct:next` should be at the `dcat:Dataset` level, alongs with a `dcat:prev` elements.